test/extended/testdata/test-imagesource-build.yaml

```diff
@@ -89,7 +89,7 @@ items:
           deploymentconfig: imagesourceapp
       spec:
         containers:
-        - image: imagesourceapp
+        - image: " "
```

Comment:
i'm afraid that'll confuse people looking at this file later (and there's no way to comment), so i'd like to keep them consistent.  I can make them both emptystring if that's the "right" way to do it.  I had thought we were using " " as the marker for "an ICT will fill this in".  Are we doing validation-fixup on both emptystring and " " ?